### ISSUE TITLE
warning for valid compressed partial WAL segment

### ISSUE THREAD

**Issue by vthriller** [postgrespro/pg_probackup#177]
Rather silly issue that is easy to reproduce:

```sh
# note the -Z option
pg_receivewal -Z 1 -D $backup/wal/$instance -h … &
pg_probackup delete -B $backup --instance $instance --delete-wal
```

Warning in question:

```
WARNING: unexpected WAL file name "00000001000003C30000006C.gz.partial"
```

I'm running PostgreSQL 11, if that matters.

**Comment by gsmol** [comment]
Hello!
True, this message is useless, we should handle ".partial" files.
Probably we should also handle partial files when restoring segments from archive.

**Comment by gsmol** [comment]
Fixed and merged into master.
Will be released with the upcoming 2.3.0 version.
